lib/aullar/styling.moon

```diff
@@ -170,6 +196,10 @@ define_class {
     ptr = @style_buffer\get_ptr(offset - 1, 1)
     style_map[ptr[0]]
 
+  get_mode_name_at: (pos) =>
+    found = @sub_style_markers\at pos
+    found and found[#found] and found[#found].name
```

Comment:
`found` will never be `nil` here by the way, no markers found will result in an empty table.